src/contracts/index.ts

```diff
@@ -0,0 +1,28 @@
+import assert from 'assert'
+import { web3 } from 'helpers/web3'
+import { BatchExchangeContract } from './BatchExchangeContract'
+import { Erc20Contract } from './Erc20Contract'
+
+function getStableConverterContract (): BatchExchangeContract {
+  const { STABLE_COIN_CONTRACT_ADDRESS } = process.env
+  assert(STABLE_COIN_CONTRACT_ADDRESS, 'STABLE_COIN_CONTRACT_ADDRESS env is required')
+
+  const stableCoinContractAddress = STABLE_COIN_CONTRACT_ADDRESS as string
```

Comment:
why is this required? shouldn't the `assert` tell TS that it's indeed a `string`?

Reply:
Perhaps cause it's the old node assert, not like the one you wrote that `asserts is string`

Reply:
good point, maybe this can be refactored so we move the assert here

Reply:
https://github.com/gnosis/dex-js/issues/29